basics/Main.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Random;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Main {
+
+  public static void pluarlize(String str, int num){
+    if(num == 0 || num > 1){
+      System.out.println("I own " + num + " " + str + "s");
+    } else {
+      System.out.println("I own " + num + " " + str);
+    }
+  }
+
+  public static void flipNHeads(int n){
+    Random rnd = new Random();
+    
+    int heads = 0;
+    int totalFlips = 0;
+    do {
+      double result = rnd.nextDouble();
+      if(result > .5){
+        System.out.println("Heads");
+        heads++;
+        totalFlips++;
+      } else {
+        System.out.println("Tails");
+        totalFlips++;
+      }
+    } while (heads < n);
+    System.out.println("It took " + totalFlips + " flips to flip " + n + " heads in a row");
+  }
+
+  public static void clock(){
+    while(true){
+      LocalDateTime now = LocalDateTime.now();
+      String time = now.format(DateTimeFormatter.ofPattern("HH:mm:ss"));
+      System.out.println(time);
+      try{
+        Thread.sleep(1000);
+      } catch(InterruptedException ex) {
+        Thread.currentThread().interrupt();
+      }
+    }
+  }
+
+  public static void main(String[] args){
```

Comment:
Putting your `main` method here is perfectly fine, however, when a class has a `main` method it is usually the first method defined in the class